node/pkg/alephium/reobserve.go

```diff
@@ -16,8 +16,14 @@ func (w *Watcher) handleObsvRequest(ctx context.Context, logger *zap.Logger, cli
 		case <-ctx.Done():
 			return
 		case req := <-w.obsvReqC:
-			assume(req.ChainId == uint32(vaa.ChainIDAlephium))
-			assume(len(req.TxHash) == 32)
+			if req.ChainId != uint32(vaa.ChainIDAlephium) {
+				logger.Error("invalid chain id, expect alephium", zap.Uint32("chainId", req.ChainId))
```

Comment:
[Nit]: s/invalid/Invalid/

Similar for other places where the first character of error message is lowercase

Reply:
Do you mean the error message should start with uppercase? Currently, the log of the guardian node code basically starts with lowercase. And it seems that the golang community prefers to start with lowercase for error messages: https://github.com/golang/go/wiki/CodeReviewComments#error-strings, although this rule does not apply to logs.

Reply:
Okay, as long as the style is consistent.